lago/cmd.py

```diff
@@ -492,47 +487,67 @@ def do_console(prefix, host, **kwargs):
 
 
 @lago.plugins.cli.cli_plugin(
-    help='Create Ansible host inventory of the environment'
+    help='Create Ansible host inventory of the environment',
+    description=dedent(
+        """
+    This method iterates through all the VMs and creates an Ansible
+    host inventory. For each vm it defines an IP address and a private key.
+
+    The default groups are based on the values which associated
+    with the following keys: 'vm-type', 'groups', 'vm-provider'.
+
+    The 'keys' parameter can be used to override the default groups,
+    for example to create a group which based on a 'service_provider',
+    --keys 'service_provider' should be added to this command.
+
+    Nested keys can be used also by specifying the path to the key, for example
+    '/disks/0/metadata/distro' will create a group based on the distro of
+    the os installed on disk at position 0 in the init file.
+    (we assume that numeric values in the path should be used as index for list
+    access).
+
+    If the value associated with the key is a list, a group will be created
+    for every item in that list (this is useful when you want to associate
+    a machine with more than one group).
+
+    The output of this command ansis printed to standard output.
+
+    Example of a possible output:
+
+    lago ansible_hosts -k 'vm-type'
+
+    [vm-type=ovirt-host]
+    lago-host1 ansible_host=1.2.3.4 ansible_ssh_private_key_file=/path/rsa
+    lago-host2 ansible_host=1.2.3.6 ansible_ssh_private_key_file=/path/rsa
+    [vm-type=ovirt-engine]
+    lago-engine ansible_host=1.2.3.5 ansible_ssh_private_key_file=/path/rsa
+
+    lago ansible_hosts -k 'disks/0/metadata/arch' 'groups'
+
+    [disks/0/metadata/arch=x86_64]
+    vm0-server ansible_host=192.168.204.3 ansible_ssh_private_key_file=/path/rsa
+    vm1-slave ansible_host=192.168.204.4 ansible_ssh_private_key_file=/path/rsa
+    vm2-slave ansible_host=192.168.204.2 ansible_ssh_private_key_file=/path/rsa
+    [groups=slaves]
+    vm1-slave ansible_host=192.168.204.4 ansible_ssh_private_key_file=/path/rsa
+    vm2-slave ansible_host=192.168.204.2 ansible_ssh_private_key_file=/path/rsa
+    [groups=servers]
+    vm0-server ansible_host=192.168.204.3 ansible_ssh_private_key_file=/path/rsa
+    """
+    )
+)
+@lago.plugins.cli.cli_plugin_add_argument(
+    '--keys',
+    '-k',
+    help='Path to the keys that should be used as groups',
+    default=['vm-type', 'groups', 'vm-provider'],
+    metavar='KEY',
+    nargs='*',
```

Comment:
It's not.
Currently OST Ansible uses ```[ovirt-host], [ovirt-engine]``` while the new names
are ```[vm-type=ovirt-host], [vm-type=ovirt-engine]```.
Since any key can be used as a group, I have to keep it this way (in order to create a unique group name).